### PR TITLE
On being 28

### PR DESCRIPTION
**Pull request by orta**
PR for my annual writeup

### CODE REVIEW

File: _posts/2014-11-21-28.markdown [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+Part of the re-branding had been that we were going to explore community events a bit more. I had ran a series of lightning talks that went beautifully called [CocoaKucha][5] ( based on the [PechaKucha][6] format ). I still don't think I've quite got the hang of being the host of an event yet. They're like this funny awkward black humour affair which doesn't always appeal. But it was a good springboard to some of the events that we played with further along the year.
+
+I headed back to the UK in December. I had been invited to a wedding in India, and that meant taking the chance to be a part of my friends first step into maturity ( his wife gave birth last week, as of writing. ) He was marring an indian lady, and they are an amazing couple. Kevin was the [roadie for my band][7] back when we [played.][8].
```
Comment (ashfurrow) [inline _posts/2014-11-21-28.markdown]: "marring" -> "marrying" ?

File: _posts/2014-11-21-28.markdown [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+[[ photos ]]
+
+We stayed in Istanbul till early may after the conference. Danger took me to a cool massage parlour for her birthday. When we were staying there were [riots][48], it gave the place a really odd atmosphere at night. 
+
+In Artsy stuff, I moved back from the [Artsy iOS App][49] to our [Gallery Portfolio app][50] to start working on version 2.0. This coincided with the first non-artsy person joining the mobile team. Ash, who I had talked to back in Amsterdam emailed me sometime asking about a job, for more info on this check his [awesome 5 year review][51]. We had gone through the interview process process and he was on his 3 month initial contract. This was double awesome, because it meant I now didn't have to concentrate on two apps at once. Plus he came with a wealth of knowledge in areas I've not touched.
```
Comment (ashfurrow) [inline _posts/2014-11-21-28.markdown]: "interview process process"

File: _posts/2014-11-21-28.markdown [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+[[ photos ]]
+
+Not one to waste time in Europe, a week or two later I headed over to Austria to see Pete Steinberger. We had made plans to go to the alps with some other iOS devs. We went, it was beautiful. We did a lot of walking, some talking and had a really casual time. 
+
+[[ photos ]]
+
+I spent a few weeks back in the UK, then headed out to NYC. For the last 3 months I had been in full email mode for a conference  I would have liked to run in NYC. I’m pretty lucky to know a lot of interesting people, and have easy access to venues over there. I wanted to explore what a conference aimed at beginners should be like. There is a full writeup on my [blog][59]. It went well, and I can’t thank the speakers enough. For most of them it was their first time! =)
+
+At the same time as this we were starting to begin production on Artsy’s next iOS app, Eidolon. I pushed the initial commit on the 4th of August and for the first month I was totally hands off, only merging PRs and quick fixes. It was an test case for us on open-sourcing an iOS actively built application and Ash was going to be the driving force on the app.
```
Comment (ashfurrow) [inline _posts/2014-11-21-28.markdown]: "an test case"

**Comment by orta** [comment]
merging, to test HTTPS things on production.

**Pull request merged**